demo/func/host.json

```diff
@@ -2,6 +2,6 @@
 	"version": "2.0",
 	"extensionBundle": {
 		"id": "Microsoft.Azure.Functions.ExtensionBundle",
-		"version": "[2.*, 3.0.0)"
+		"version": "[4.0.0, 5.0.0)"
```

Comment:
There's a external Azure function in this folder that also needs to be converted to the v4 model